packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -57,9 +58,15 @@ class AnalyzeCommand extends FlutterCommand {
   @override
   Future<Null> runCommand() {
     if (argResults['watch']) {
-      return new AnalyzeContinuously(argResults, runner.getRepoPackages()).analyze();
+      return new AnalyzeContinuously(
+        argResults, runner.getRepoPackages(), previewDart2: argResults['preview-dart-2']
+      ).analyze();
     } else {
-      return new AnalyzeOnce(argResults, runner.getRepoPackages(), workingDirectory: workingDirectory).analyze();
+      return new AnalyzeOnce(
+        argResults, runner.getRepoPackages(),
```

Comment:
Nit: can you put these on a separate line

Reply:
👍